route/build/src/irf_route.f90

```diff
@@ -190,6 +195,27 @@ subroutine segment_irf(&
                    RCHFLX_out(iens,segIndex)%BASIN_QR(1),RCHFLX_out(iens,segIndex)%REACH_Q_IRF
   endif
 
+  ! print statement to compare the computed REACH_Q_IRF and water management abstraction/injection
+  ! print*, NETOPO_in(segIndex)%REACHID, RCHFLX_out(iens,segIndex)%REACH_Q_IRF, RCHFLX_out(iens,segIndex)%REACH_WM_FLUX
+
+  ! take out the water from the reach if the wm flag is true and the value are not missing
+  ! here we should make sure the real missing is not injection (or negative abstration)
+  if((RCHFLX_out(iens,segIndex)%REACH_WM_FLUX /= realMissing).and.(is_flux_wm)) then
```

Comment:
some style thing.  2 indentations for if statement, do loop etc?  Looks like you are using 2 indentations (instead of single indentation) for the rest of your code. and I prefer 2 indentation as well.

Reply:
I personally like 2 indentations however I usually follow what is in the file. in this file there is 1 indentations so I was following that. fixed